addons/point_of_sale/migrations/10.0.1.0.1/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+column_copies = {
+    'pos_config': [
+        ('state', None, None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.copy_columns(env.cr, column_copies)
```

Comment:
No need to copy the column, as the column is not dropped, so you can use it as is. We only copy columns, when we need to make changes over the same column (for example, if there are changes on the values of a selection field).